spell/spell-reviewer-goerli-checklist.md

```diff
@@ -12,6 +12,7 @@ Spell Actions:
   * [ ] Off (default, spell casted after deploying, pause delay 60 sec.)
 * [ ] Exec Hash
   * [ ] empty (default)
+  * [ ] ensure `description` matches `Goerli Spell`
```

Comment:
I would say 'is' rather than 'matches' here but this is good

Reply:
18cf91c